### ISSUE TITLE
When installed on Magento 2.1.6 the Add product Shows ERROR

### ISSUE THREAD

**Issue by ebofi** [Talv/magento2-wysiwyg-fix#1]
We are also experiencing the Magento WYSIWYG issue on image url, When uploading the image via editor and after we log out of the admin, The images become 404 . When checking the url of the image its pointing to admin url of the system.

So i had tried to install this plugin, but after i install the plugin it shows error on website.

**Comment by ebofi** [comment]
Update : After installing the plugin when i run   php bin/magento setup:di:compile

This is what i get in  
Interceptors generation... 4/7 [================>-----------]  57% 15 secs 174.0 MiB
[InvalidArgumentException]
  Plugin class Vendor\Module\Plugin\WysiwygConfig doesn't exist



setup:di:compile

**Comment by Talv** [comment]
> Plugin class Vendor\Module\Plugin\WysiwygConfig doesn't exist

FQN doesn't look correct.. are you sure you've installed module correctly? Because it apparently tries to reference ```Vendor\Module\Plugin\WysiwygConfig``` which indeed doesn't exists, as correct FQN is ```Talv\WysiwygFix\Plugin\WysiwygConfig```.

Did you install it via composer as README says or some other way?

**Comment by ebofi** [comment]
yes i followed the Instruction you provided in the plugin installation section. I am using Magento 2.1.6

**Comment by ebofi** [comment]
UPDATE 👍 

I tried again installing the plugin and the site shows error and i had enabled the debug mode and the error shows

1 exception(s):
Exception #0 (InvalidArgumentException): Plugin class Vendor\Module\Plugin\WysiwygConfig doesn't exist

Exception #0 (InvalidArgumentException): Plugin class Vendor\Module\Plugin\WysiwygConfig doesn't exist

**Comment by Talv** [comment]
The prefix it outputs ```Vendor\Module``` doesn't make sense. This plugin doesn't use such prefix, thus there's apparently something else getting in the way. It's an issue on your end, as plugin works fine on clean install of 2.1.6. I might not be able to help.

But you could try alternative installation method. Instead of using composer, just download & dump content of this repo into: ```/app/code/Talv/WysiwygFix```. Although, before you do that remove the copy installed by composer - so they won't be in conflict.

**Comment by ebofi** [comment]
ok let me try it
